api/job/v1beta1/job.proto

```diff
@@ -63,19 +63,13 @@ message GetAvailableJobsRequest {
   // Should be used consistently across all APIs.
   string area_key = 1;
   string worker_id = 2;
-  double range_limit = 3;
```

Comment:
Don't we still need `range_limit` to constrain our geo-search? As I understand, _range limit_ would be something like 1 mile, and the search would return any drivers within 1 mile of the pickup location.

Reply:
yes we need. Its limit. 
was renamed from range_limit, but i am thinking of better name `max_distance_meters` ? .
better naming suggestions are welcome.

Reply:
`radius_meters` is 👍🏼